README.md

```diff
@@ -24,13 +24,13 @@ libraryDirectory := file("/usr/lib64")
 point to your xdg-shell-unstable-v6.xml, your include path, and the path that contains the .so files on your system.
 
 ### Note for Part 3
-I had to add the file xdg-shell-unstable-v6-protocol.h from a manual build of wlroots to get this step able to work.
-My distribution's wlroots-devel package does not contain this file, despite the wlr_xdg_shell_v6.h header and depending
-on this protocol
+The file, xdg-shell-unstable-v6-protocol.h, from a manual build of wlroots was necessary for this step to function correctly.
```

Comment:
This isn't actually necessary anymore, the build system generates the xdg-shell-unstable-v6-protocol.h file automatically on compile. Please go ahead and remove this note entirely as it's no longer relevant.